### PR TITLE
Add python script and readme

### PR DESCRIPTION
**Pull request by selamanse**
This change introduces a python script that applies a lifecycle rule to a s3 bucket to delete data older than 30 days.

closing #7

### CODE REVIEW

File: python_delete_s3_data/Readme.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+# python script to delete s3 data
+
+this project holds a python script which can delete S3 data which is older than 30 days
```
Comment (DiptoChakrabarty) [inline python_delete_s3_data/Readme.md:3]: can you also change this description to just applying lifecycle policy s3

**Comment by DiptoChakrabarty** [comment]
thanks for the PR @selamanse can you rename the directory to aws/apply_lifecycle_policy_s3 

I am looking for something different for which I think I have to update the issue but this code looks good but I just want to accept it in another directory.

Is that fine by you?

**Comment by selamanse** [comment]
thanks for the quick feedback @DiptoChakrabarty. sure thing I renamed / relocated the script as you desired. can you check please if this version ok?

**Comment by DiptoChakrabarty** [comment]
thanks a lot

**Pull request merged**